ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectSSOTest.kt

```diff
@@ -78,21 +70,18 @@ class RedirectSSOTest : StringSpec() {
 
             // Get response from AuthnRequest
             val response = sendRedirectAuthnRequest(queryParams)
-            BindingVerifier.verifyHttpStatusCode(response.statusCode)
-
-            val idpResponse = getServiceProvider(IdpSSOResponder::class)
-                    .getRedirectResponse(response).decorate().apply {
+            val finalHttpResponse =
+                    getServiceProvider(IdpSSOResponder::class).getResponseForRedirectRequest(
+                            response)
+//            SingleSignOnProfileVerifier.verifyBinding(finalHttpResponse)
```

Comment:
✏️ Should probably include a todo comment here indicating why this is commented out and when/how it will be enabled.

Reply:
I'm going to leave it because Blen's PR uncomments this, so it will get taken care of when I rebase after hers gets merged.